app/models/lock_version.rb

```diff
@@ -41,8 +41,9 @@ def increment!
 
 private
 
-  def content_item_target?
-    target_type == 'ContentItem'
+  def edition_target?
+    # The 'Edition' class used to be called the 'ContentItem' class.
```

Comment:
once the Editions stuff is stable, will we migrate these polymorphic fields? if so, let's try and make that explicit (or at least have a clear plan for it). otherwise we run the risk of generating enduring confusion...